release-process.md

```diff
@@ -186,6 +186,18 @@ that looks something like `[VOTE][RESULT] ...`.
 
 <h3>Finalize the release</h3>
 
+Note that `dev/create-release/do-release-docker.sh` script (`finalize` step ) automates most of the following steps **except** for:
+- Publish to CRAN
```

Comment:
PyPI?

Reply:
"Uploading PySpark to PyPi" is automated.

Reply:
Hm? Last time I did release I remember I still need to do `twine upload`, has it changed?